nginx/https_arcsi.conf.template

```diff
@@ -44,12 +44,15 @@ server {
 
     location / {
         proxy_http_version 1.1;
+        proxy_max_temp_file_size 0;
+        chunked_transfer_encoding off;
+        proxy_buffering off;
+        proxy_request_buffering off;
```

Comment:
Are these lines (47-50) related to the CIRS issue too?

Reply:
@pvj says: these are limitations that are recommended for a restricted CORS setting. See nginx specs for more details.